### PR TITLE
Link to style guide

### PR DESCRIPTION
**Pull request by arthurevans**

### CODE REVIEW

**Comment by arthurevans** [comment]
@katejeffreys ping?

File: STYLE.md [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
+we've made in the course of developing the Polymer documentation.
 
-There are counterexamples for each of these rules in the existing docs.
+As a general rule, the Polymer documentation tries to follow the 
+[Google Developer Documentation Style Guide](https://developers.google.com/style/)
+(GDDSG).
+
+This document:
+
+-  Summarizes a some common points of developer documentation style, which
```
Comment (katejeffreys) [inline STYLE.md]: "Summarizes a some"

**Comment by katejeffreys** [comment]
Sorry for the delay. 1 typo, otherwise lgtm

**Comment by arthurevans** [comment]
Typo in the style guide—extra points. D'oh 🗡

**Pull request merged**